haystacker-server/src/main/kotlin/net/sr89/haystacker/server/filter/ExceptionHandlingFilter.kt

```diff
@@ -29,6 +26,9 @@ class ExceptionHandler(val next: HttpHandler) : HttpHandler {
         } catch (e: InvalidHslOperatorException) {
             Response(BAD_REQUEST)
                 .with(stringBody of "Invalid operator (${e.operator}) for symbol '${e.symbol}' and value '${e.value}'")
+        } catch (e: InvalidTaskIdException) {
+            Response(BAD_REQUEST)
+                    .with(stringBody of "Provided Task ID is invalid : ${e.taskId}")
```

Comment:
`The` missing in front of `provided`. Or better yet, `Invalid Task ID provided: ${e.taskId}`